src/index.css

```
@@ -95,4 +96,28 @@
     background-color: #fffbf6;
     box-sizing: border-box;
   }
+
+@layer utilities {
+  .no-scrollbar {
+    -ms-overflow-style: none; 
+    scrollbar-width: none;    
+  }
+
+  .no-scrollbar::-webkit-scrollbar {
+    display: none;            
+  }
+
+  @keyframes quote-flip {
```

`@theme` 안에 작성해주시면 될 것 같습니다!

```suggestion
--animate-quote-flip: quote-flip 0.4s ease both;

  @keyframes quote-flip {
    0% {
      opacity: 0;
    }
    100% {
      opacity: 1;
    }
  }
```

---

넵! 반영했습니다!